neuromation/api/jobs.py

```diff
@@ -729,37 +790,112 @@ def _container_from_api(data: Dict[str, Any], parse: Parser) -> Container:
     )
 
 
-def _container_to_api(container: Container, config: Config) -> Dict[str, Any]:
+def _job_to_api(
+    config: Config,
+    image: RemoteImage,
+    entrypoint: Optional[str] = None,
+    command: Optional[str] = None,
+    working_dir: Optional[str] = None,
+    http: Optional[HTTPPort] = None,
+    env: Optional[Mapping[str, str]] = None,
+    volumes: Sequence[Volume] = (),
+    secret_env: Optional[Mapping[str, URL]] = None,
+    secret_files: Sequence[SecretFile] = (),
+    disk_volumes: Sequence[DiskVolume] = (),
+    tty: bool = False,
+    shm: bool = False,
+    name: Optional[str] = None,
+    preset_name: Optional[str] = None,
+    tags: Sequence[str] = (),
+    description: Optional[str] = None,
+    pass_config: bool = False,
+    wait_for_jobs_quota: bool = False,
+    schedule_timeout: Optional[float] = None,
+    restart_policy: JobRestartPolicy = JobRestartPolicy.NEVER,
+    life_span: Optional[float] = None,
+) -> Dict[str, Any]:
     primitive: Dict[str, Any] = {
-        "image": _as_repo_str(container.image),
-        "resources": _resources_to_api(container.resources),
+        "container": _container_to_api(
```

Comment:
Maybe the new REST API can use a flat JSON structure, as we do for Python API in this PR?
Other guys like JavaScript devs will say "thank you" for the change, sure!